File: Tyche.iOS/Pool/Sources/Pool/Creator/CreatePoolModelTransformer.swift
import DataPool

extension CreatePoolModel {
    func toCreatePoolInput(ownerGamblerId: String) -> CreatePoolInput {
        CreatePoolInput(
            poolLayoutId: poolLayoutId,
            poolName: poolName,
            ownerGamblerId: ownerGamblerId
        )
    }
}

File: Android/pool/src/main/java/com/felipearpa/tyche/pool/creator/CreatePoolModelTransformer.kt
package com.felipearpa.tyche.pool.creator

import com.felipearpa.tyche.data.pool.domain.CreatePoolInput
import com.felipearpa.tyche.data.pool.type.PoolName

internal fun CreatePoolModel.toCreatePoolInput(ownerGamblerId: String) =
    CreatePoolInput(
        poolLayoutId = poolLayoutId,
        poolName = PoolName(poolName),
        ownerGamblerId = ownerGamblerId,
    )
